src/nyc_trees/nyc_trees/settings/base.py

```diff
@@ -70,6 +72,19 @@
 # END CACHE CONFIGURATION
 
 
+# CELERY CONFIGURATION
+BROKER_URL = 'redis://{0}:{1}/2'.format(
+    environ.get('NYC_TREES_CACHE_HOST', 'localhost'),
+    environ.get('NYC_TREES_CACHE_PORT', 6379))
+
+CELERY_ACCEPT_CONTENT = ['json']
+CELERY_TASK_SERIALIZER = 'json'
```

Comment:
In the past we've always used `pickle` because we don't communicate with other languages and don't accept tasks from untrustworthy sources. It's faster and supports more data types.
http://celery.readthedocs.org/en/latest/userguide/calling.html#serializers

I'm not sure of the reach of what we'll lose, feature-wise, but we should keep pickle serialization in mind if it becomes a problem.